Makefile

```diff
@@ -68,3 +68,6 @@ APP_SOURCE_PATH += src
 #DISABLE_STANDARD_BAGL_UX_FLOW = 1
 
 include $(BOLOS_SDK)/Makefile.standard_app
+
+configure:
```

Comment:
We want to keep the makefile as simple as possible. Is it possible to do that outside the makefile (especially if it is linked to tool configuration) ?

Reply:
It's dumping the DEFINES variable that is only available in the makefile, maybe have the makefile export it to en env variable and then convert this variable to a .h file?

Reply:
Did a fix to move all extensions to the makefile to .vscode folder to avoid polluting main makefile